api/apps/brand_partners/model.coffee

```diff
@@ -0,0 +1,83 @@
+#
+# Library of retrieval, persistance, validation, json view, and domain logic
+# for the "brand-partners" resource.
+#
+
+_ = require 'underscore'
+db = require '../../lib/db'
+async = require 'async'
+Joi = require 'joi'
+Joi.objectId = require 'joi-objectid'
+{ ObjectId } = require 'mongojs'
+{ ARTSY_URL, API_MAX, API_PAGE_SIZE } = process.env
+
+#
+# Schemas
+#
+schema = (->
+  id: @objectId()
+  partner_id: @objectId()
+  slug: @string().allow('', null)
```

Comment:
So in that case we can just do `query = { _id: ObjectId(id) }` b/c we don't need to support `/api/brand_partners/:slug` we should instead just query based on the partner id (which you support [here](https://github.com/kanaabe/positron/blob/brandpartnership/api/apps/brand_partners/model.coffee#L34) but I think you need to add a line that converts string -> object id [here](https://github.com/kanaabe/positron/blob/brandpartnership/api/apps/brand_partners/model.coffee#L48)) e.g. `/api/brand_partners?partner_id=:id`.